src/amo/sagas/reviews.js

```diff
@@ -281,21 +281,23 @@ function* manageAddonReview(
       yield put(hideEditReviewForm({ reviewId: reviewFromResponse.id }));
     }
 
-    invariant(reviewFromResponse.version, 'version is required');
-    yield put(
-      setLatestReview({
-        addonId: reviewFromResponse.addon.id,
-        addonSlug: reviewFromResponse.addon.slug,
-        review: reviewFromResponse,
-        userId: reviewFromResponse.user.id,
-        versionId: reviewFromResponse.version.id,
-      }),
-    );
+    if (!reviewFromResponse.is_developer_reply) {
```

Comment:
> it still makes sense to me that we wouldn't dispatch either of those actions for a reply update as neither are needed.

Yes, I agree. I see it now. I guess `fetchAddon` would be possible since it doesn't require `version` but I can't think of why we need it.